src/tests/ftest/util/telemetry_utils.py

```diff
@@ -515,6 +515,12 @@ class TelemetryUtils():
         ENGINE_IO_OPS_TGT_UPDATE_ACTIVE_METRICS +\
         ENGINE_IO_OPS_UPDATE_ACTIVE_METRICS
     ENGINE_NET_METRICS = [
+        "engine_net_delay_stddev",
+        "engine_net_glitch",
+        "engine_net_delay_max",
```

Comment:
just a note here that max can be a bit misleading, as i believe right after engine format we experience a big swim processing delay, likely due to the threads being starved. we might want to filter that max out to get an accurate value during the rest of the runtime